google/cloud/spanner_v1/pool.py

```diff
@@ -229,12 +260,24 @@ def get(self, timeout=None):
         if timeout is None:
             timeout = self.default_timeout
 
+        start_time = time.time()
+        current_span = get_current_span()
+        span_event_attributes = {"kind": type(self).__name__}
+        current_span.add_event("Acquiring session", span_event_attributes)
         session = self._sessions.get(block=True, timeout=timeout)
+        span_event_attributes["session.id"] = session.session_id
 
         if not session.exists():
+            current_span.add_event(
+                "Session is not valid, recreating it", span_event_attributes
+            )
             session = self._database.session()
             session.create()
+            span_event_attributes["id"] = self._session_id
+            current_span.add_event("Using Session", span_event_attributes)
 
+        span_event_attributes["time.elapsed"] = time.time() - start_time
+        current_span.add_event("Acquired session", span_event_attributes)
```

Comment:
It makes more sense to have "Acquired Session" and then "Using Session".

Also I feel "Acquired Session" event should exist when a session lies in pool. If there is no session in pool then we are creating session and in that case "Acquired Session" does not make sense.

So the span hierachy would something be like this.

If session exists in the pool, then
1. Acquiring Session
2. Acquired Session
3. Using Session

If no session in the pool 
1. Acquiring Session
2. No Session Available
3. Creating Session
4. CloudSpanner.CreateSession
5. Using Session

Does that sound good?
@surbhigarg92 Do you have any comments on this structure?